swap_meet/vendor.py

```diff
@@ -1,2 +1,83 @@
+from .item import Item
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
```

Comment:
We can use a ternary here:

```suggestion
        self.inventory = inventory if inventory is not None else []
```